### ISSUE TITLE
Not receiving full options

### ISSUE THREAD

**Issue by dfischer** [acemtp/meteor-accounts-slack#5]
My user object is only being populated with `services.slack.id` and `services.slack.accessToken` how to get all the other ones defined here:

https://github.com/acemtp/meteor-accounts-slack/blob/master/slack_server.js

?

**Comment by srbartlett** [comment]
Running into the same issue.  Did you find a solution?

**Comment by dfischer** [comment]
Yeah I forked the package `meteor add dfischer:accounts-slack` and I fill out a lot more details now

**Comment by acemtp** [comment]
Hi!
@dfischer if you would like to PR your changes, I'll be happy to merge them and publish a new version.

**Comment by dfischer** [comment]
I overloaded it quite a bit I think – not sure if everyone wants what I did. If you think it's appropriate then I'll be happy to do so: https://github.com/dfischer/meteor-accounts-slack/commit/ebc8f872fca5a73b1c95b0142722ada9714eba26 @acemtp (I'll also put back the html file)
